src/Core/src/Handlers/SearchBar/SearchBarHandler.iOS.cs

```diff
@@ -23,35 +24,14 @@ protected override MauiSearchBar CreatePlatformView()
 
 		protected override void ConnectHandler(MauiSearchBar platformView)
 		{
-			platformView.CancelButtonClicked += OnCancelClicked;
-			platformView.SearchButtonClicked += OnSearchButtonClicked;
-			platformView.TextSetOrChanged += OnTextPropertySet;
-			platformView.OnMovedToWindow += OnMovedToWindow;
-			platformView.ShouldChangeTextInRange += ShouldChangeText;
-			platformView.OnEditingStarted += OnEditingStarted;
-			platformView.EditingChanged += OnEditingChanged;
-			platformView.OnEditingStopped += OnEditingStopped;
+			_proxy.Connect(this, VirtualView, platformView);
```

Comment:
LGTM.

Reply:
This PR completely breaks all the UI tests, which accesses a `SearchBar` to find & run each test?

I did not figure out the cause, it seemed like `SearchBar` generally worked when I tried in other apps like the DeviceTests.